app/src/main/java/com/hover/stax/SplashScreenActivity.java

```diff
@@ -30,29 +40,74 @@
 	@Override
 	protected void onCreate(@Nullable Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
+		startSplashForegroundSequence();
+		startBackgroundProcesses();
+	}
 
-//		if (SelfDestructActivity.isExpired(this)) {
-//			startActivity(new Intent(this, SelfDestructActivity.class));
-//			finish();
-//			return;
-//		}
+	private void startSplashForegroundSequence() {
+		initSplashAnimation();
+		authenticateBiometricOrNavigateScreenAfter110sec();
+	}
 
-		Amplitude.getInstance().initialize(this, getString(R.string.amp)).enableForegroundTracking(getApplication());
+	private void startBackgroundProcesses() {
```

Comment:
Best if initialized on the application class

Reply:
That is initializing
* Amplitude
* Hover
* Notification Channel
* Workers - if it's work manager

Reply:
Yea, one or two of these APIs doesn't work well when initialized in the Application class. An example is the Hover SDK itself.

Reply:
Makes sense - You could try to investigate why this happens on the Hover SDK side (On my Interview solution, I initialized the HoverSDK on the application class) 

But the rest would make sense if they were initialized on the Application Class.